components/brave_rewards/resources/rewards_panel/components/panel_overlays.tsx

```diff
@@ -97,27 +101,29 @@ export function PanelOverlays () {
 
   if (onboardingResult || !rewardsEnabled || needsCountry) {
     const onHideResult = () => {
+      if (onboardingResult === 'success') {
+        setTimeout(() => {
+          tabOpener.openTab(urls.rewardsTourURL)
+        }, 2000)
+      }
       setOnboardingResult(null)
     }
 
     const onEnable = (country: string) => {
       host.enableRewards(country).then((result) => {
         setOnboardingResult(result)
-        if (!rewardsEnabled && result === 'success') {
-          setOnboardingResult(null)
-          setShowTour(true)
-        }
       })
     }
 
     return (
       <RewardsOptInModal
         availableCountries={availableCountries}
         defaultCountry={defaultCountry}
-        initialView={needsCountry ? 'declare-country' : 'default'}
+        initialView={needsCountry ?
```

Comment:
Since this isn't really a modal anymore, I'm pondering whether we should make this a regular (non-modal) component and put it in `panel.tsx` instead. It would render instead of the regular panel content if necessary. 🤔

Reply:
Done in https://github.com/brave/brave-core/pull/18717/commits/bea29900fbb13979989c3057bf3e0772602ec2ac